daemon/lightning/lnd/lnd.go

```diff
@@ -72,34 +85,67 @@ var (
 	Testnet Network = "testnet"
 )
 
-// NewClient creates a lnd client from a daprlndconnectURI string.
-// This Client establishes a grpc connection with a lnd node using dapr.
-// It is using two stubs: router and ln.
+var ErrMutuallyExclusiveOptions = errors.New("LNDConnect is mutually exclusive with LndEndpoint, MacaroonFilePath and TLSCertFilePath")
```

Comment:
nit: maybe we should return a shorter error. Then create the error for the grpc with this long message and the cli with a diferent one. Since for the cli we should return `--lndconnect is mutually exclusive with --lnd-host, --macaroon and -tls-cert`

Reply:
Done, did a shorter one